### DISCUSSION TITLE
看看你的帐户是不是被ban了，看下邮件。然后看下API用量是否有余额：https://platform.openai.com/account/usage

### DISCUSSION THREAD

**Discussion by Evan-tc** [Yue-Yang/ChatGPT-Siri#83] [General]
看看你的帐户是不是被ban了，看下邮件。然后看下API用量是否有余额：https://platform.openai.com/account/usage

_Originally posted by @Yue-Yang in https://github.com/Yue-Yang/ChatGPT-Siri/issues/80#issuecomment-1483695517_

**Comment by Evan-tc** [comment]
![image](https://user-images.githubusercontent.com/40584291/227699971-c6c82cb5-cc32-49fe-9637-2e018dbb9350.jpg)
看这个应该是有的吧 ，但是还是用魔法进不去

**Reply by Yue-Yang** [reply]
你网络问题自己解决啊，别人怎么帮啊

**Reply by Evan-tc** [reply]
但是我魔法能直接进chatgpt网页的
